### ISSUE TITLE
incomplete file set for v3.50-24

### ISSUE THREAD

**Issue by dsrawding** [kwindrem/GuiMods#278]
Can't install v10.66 due to incomplete file set for v3.50-24 error

**Comment by kwindrem** [comment]
Major changes from Victron. Working on an update but it will take some time.

**Comment by kwindrem** [comment]
should be fixed in v10.67

**Comment by dsrawding** [comment]
Updated to v10.67 and still getting the same error unfortunately.

**Comment by dsrawding** [comment]
I removed from the gui, logged into the cli and removed the directory.  Then downloaded the code from GitHub and still got the same error.  Looked in the FileSet directory and both ~24 and ~25 were there.  System was still trying to install from ~24.  Removed ~24 and got it going.  

Stange though as I then downloaded the code again a couple minutes later from GitHub and ~24 was gone.  You were probably working your magic.  Anyway I'm up and running!
